packages/blade-old/scripts/icon-migrate-codemod/transforms/blade-old-icon.ts

```diff
@@ -0,0 +1,84 @@
+import iconMap from "../iconMap";
+import {

```

Comment:
why create this as a new file? We are not going to use it elsewhere so why not put the content of this file under `cli.js` itself?

Reply:
cli.js independent from the codemod, cli.js is the entrypoint to execute the `jscodeshift` 

jscodeshift works by passing a transformer file like so: 

```
jscodeshift -t ./transforms/blade-old-icon.ts
```